proxy/src/main.rs

```diff
@@ -9,15 +9,18 @@ use anyhow::bail;
 use clap::{App, Arg};
 use state::{ProxyConfig, ProxyState};
 use std::thread;
+use zenith_utils::http::endpoint;
 use zenith_utils::{tcp_listener, GIT_VERSION};
 
 mod cplane_api;
+mod http;
 mod mgmt;
 mod proxy;
 mod state;
 mod waiters;
 
 fn main() -> anyhow::Result<()> {
+    zenith_metrics::set_common_metrics_prefix("proxy");
```

Comment:
@bojanserafimov 
btw, I suggest use `zenith_proxy` as metrics prefix to avoid any mix up with other products, As example [prometheus-proxy](https://github.com/pambrose/prometheus-proxy) has the same prefix `proxy`